docker_reduction/mantid/tests/input/load_script.py

```diff
@@ -8,21 +8,115 @@
 Used to test that a file can be loaded into mantid from the input directory and
```

Comment:
Can you put this file back to the original test script that was in here (I am guessing the tests will fail due to this)